pkg/apis/cr/v1alpha1/repositoryserver_types.go

```diff
@@ -76,6 +76,14 @@ type Repository struct {
 	Hostname string `json:"hostname"`
 	// PasswordSecretRef has the password required to connect to kopia repository
 	PasswordSecretRef corev1.SecretReference `json:"passwordSecretRef"`
+	CacheSizeSettings CacheSizeSettings      `json:"cacheSizeSettings"`
```

Comment:
Should we set omitEmpty here or mark it as required in CRD?

Reply:
added it